genetics_ukbb/phasediff.json

```diff
@@ -0,0 +1,4 @@
+{
+   "EchoTime1": 0.0,
+   "EchoTime2": 0.001
```

Comment:
did you make these values up @rwblair ?

what's keeping us from finalizing this PR?

Reply:
yup these are made up. Checked with  dataset owner and these were not recorded when the dataset was created. There has to be a non zero difference between them to allow validator to pass.

I made one other deviation from the original dataset data to make it bids compatible for the example, Updated genetics_ukbb/task-FaceShapeEmotion_bold.json genetics_ukbb/task-rest_bold.json repition time to be 7.35 instead of 0.735 since there were slice timings upwards of 5 seconds. @sappelhoff You ok with this, I figured examples should be closer to the ideal of the standard than the idiosyncrasies of the datasets (that conflict with the standard) the examples were derived from.

Reply:
> yup these are made up. Checked with dataset owner and these were not recorded when the dataset was created. There has to be a non zero difference between them to allow validator to pass.

is there no way to re-compute the values from the data?

>  Updated genetics_ukbb/task-FaceShapeEmotion_bold.json genetics_ukbb/task-rest_bold.json repition time to be 7.35 instead of 0.735 since there were slice timings upwards of 5 seconds.

so 0.735 was probably a typo?

--- 

I think we should try to not include made-up values. Worst case, could we include an exception for datasets where we have missing data through the bids-validator config :thinking: